### ISSUE TITLE
Add a way to disable feed generation (for development)

### ISSUE THREAD

**Issue by chauncey-garrett** [octopress/feeds#19]
If I enable the tags feed, generation of my site increases from less than 5 s to around half a minute (there's 80-ish different tags). I'm OK with this for production but for development that's way to long.

In my dev workflow, I keep two `_config.yml` files: 
1. one for development and general settings
2. another for production, which overrides certain settings in the first for deploy

This has worked well for me, but AFAIK for Octopress/feeds (and perhaps other similar plugins?) there doesn't seem to be a way to do something similar.

In short, it's be nice to have a on/off switch for feeds that could be configured in the site `_config.yml` file. I imagine this would be useful for troubleshooting as well.
